osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs

```diff
@@ -59,6 +59,8 @@ public class DrawableRoomPlaylistItem : OsuRearrangeableListItem<PlaylistItem>
 
         public readonly PlaylistItem Item;
 
+        public Drawable RemoveButton { get; private set; }
```

Comment:
Is this only exposed for testing purposes? Not sure about that..

Reply:
It is, any suggestions? I've just taken the easiest path. I don't want to `.ChildrenOfType<Button>.ElementAt(x)`.

Reply:
Check the icon i guess? and make a local helper property to grab it in the test class.

Reply:
There's like... 4 test classes doing this. But okay.

Reply:
I can't do it, it's too flimsy. I've added back the class in a minimal state. If you don't agree with this then I suggest you make the change yourself.

Reply:
I think that's fine, not sure why it was removed in the first place.

Reply:
Because there's going to be another dummy class, and that just seems silly in comparison to exposing the buttons.